src/doc/manifest.md

```diff
@@ -451,13 +451,19 @@ To structure your code after you've created the files and folders for your proje
 Files located under `examples` are example uses of the functionality provided by
 the library. When compiled, they are placed in the `target/examples` directory.
 
-They must compile as executables (with a `main()` function) and load in the
```

Comment:
Perhaps this paragraph could be:

> They can compile as executables (with a `main()` function) or libraries and pull in the main library through `extern crate <library-name>`. Examples are compiled when you run your tests to prevent them from bitrotting.